networkit/cpp/Unittests-X.cpp

```diff
@@ -1,81 +1,58 @@
-//============================================================================
-// Name        : Unittests-X.cpp
-// Author      : Christian Staudt (christian.staudt@kit.edu),
-//		 Henning Meyerhenke (henning.meyerhenke@kit.edu)
-// Version     :
-// Copyright   : 2012, Christian Staudt, Henning Meyerhenke
-// Description : Calling unit tests and benchmarks
-//============================================================================
+/*
+ * Unittests-X.cpp
+ *
+ *  Created on: 2012
+ *      Author: Christian Staudt <christian.staudt@kit.edu>,
+ *              Henning Meyerhenke <henning.meyerhenke@kit.edu>,
```

Comment:
I guess that these email addresses are quite outdated, can we do something about this?

Reply:
My suggestion would be the following: Leave them in for the moment and in near future reformat all headers to have consistent layout and information. For instances, some headers state `cls` or `henning`. Then we can keep the mail addresses at one central place where all authors are listed.

Reply:
Sounds reasonable.